extensions/kind/src/create-cluster.ts

```diff
@@ -124,8 +137,15 @@ export async function createCluster(
     ingressController = params['kind.cluster.creation.ingress'] === 'on';
   }
 
+  // grab custom kind node image if defined
+  let extraConfig = '';
+  const image = params['kind.cluster.creation.image'];
+  if (image !== undefined && image.length > 0) {
+    extraConfig = 'image: ' + image;
```

Comment:
controllerPlaneImage = image (not a generic extraConfig as anyway it's extra only for the contro plane where you inserted it)